components/external_server/src/routes/metric.py

```diff
@@ -138,3 +139,31 @@ def post_metric_attribute(metric_uuid: MetricId, metric_attribute: str, database
     if metric_attribute in ATTRIBUTES_IMPACTING_STATUS and (latest := latest_measurement(database, metric)):
         return insert_new_measurement(database, latest.copy())
     return dict(ok=True)
+
+
+@bottle.post("/api/v3/metric/<metric_uuid>/issue/new", permissions_required=[EDIT_REPORT_PERMISSION])
+def add_metric_issue(metric_uuid: MetricId, database: Database):
+    """Add a new issue to the metric using the configured issue tracker."""
+    data_model = latest_datamodel(database)
+    reports = latest_reports(database, data_model)
+    report = latest_report_for_uuids(reports, metric_uuid)[0]
+    issue_tracker = instantiate_issue_tracker(report)
+    metric, subject = report.instance_and_parents_for_uuid(metric_uuid=metric_uuid)
+    summary = f"Quality-time metric '{metric.name}'"
+    metric_url = dict(bottle.request.json)["metric_url"]
+    description = (
+        f"Please address metric '[{metric.name}|{metric_url}]' of subject '{subject.name}' "
```

Comment:
would it be useful to get target and current value (plus their units) in the description?

Reply:
Yes, the description needs improvement.

Reply:
On second thought, copying the measurement value to the issue only leads to outdated information since its not updated.